serving/operator/deploy/resources/kourier/kourier.yaml

```diff
@@ -0,0 +1,275 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kourier-system
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-logging
+  namespace: kourier-system
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-observability
+  namespace: kourier-system
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: kourier
+  namespace: kourier-system
+spec:
+  ports:
+    - name: http
```

Comment:
This should be `http2`. See me other comment on this PR.

Reply:
@mgencur Ok, I will fix it. Thanks!